content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/02-schema-incompatibilities.mdx

```diff
@@ -23,13 +23,13 @@ techMetaDescriptions:
 
 ## Overview
 
-Each section on this page describes a potential problem when upgrading from Prisma 1 to Prisma 2.*x* and later and explains the available workarounds.
+Each section on this page describes a potential problem when upgrading from Prisma 1 to Prisma 2._x_ and later and explains the available workarounds.
```

Comment:
```suggestion
Each section on this page describes a potential problem when upgrading from Prisma 1 to Prisma 2._x_ and later, and explains the available workarounds.
```

IMO the additional comma makes the sentence easier to parse.

Reply:
Hm, it looks like these changes were actually introduced by the autoformatter and slipped in by mistake. So thanks for commenting on them!

I'll revert them to avoid muddling the PR, so I'll leave the comma suggestion (agree that it does read better)